### PR TITLE
Added Subunit inventory field type

### PR DESCRIPTION
**Pull request by davide-alghi**
## Description
Subunit inventory field type is missed

## Testing
In BE layout editor, select one inventory module with Subunit field included, remove the Subunit field, try to re-add Subunit field: it's not available.
Subunit field is set in database by installation queries.

## Notes
Where **Unit** and **Subunit** fields are set, it would be appropriate to change field type to, respectively:

- Unit to Unit
- Subunit to Subunit

Now both have Value type in all YF built-in inventory modules
Change needs to be done via service pack for all ***_invfield** tables, also including custom inventory modules
_Added some info below in comments_

## Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [ ] My code is written according to the guidelines found [here] (https://github.com/php-fig/fig-standards).
- [ ] I have signed the Contributor Licence Agreement between me and YetiForce.

### CODE REVIEW

File: modules/Vtiger/inventoryfields/Subunit.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * Inventory Subunit Field Class.
+ *
+ * @package   InventoryField
+ *
+ * @copyright YetiForce Sp. z o.o
+ * @license   YetiForce Public License 3.0 (licenses/LicenseEN.txt or yetiforce.com)
+ * @author    Arkadiusz Adach <a.adach@yetiforce.com>
```
Comment (mariuszkrzaczkowski) [inline modules/Vtiger/inventoryfields/Subunit.php]: This can be changed to your data :)

Comment (davide-alghi) [reply modules/Vtiger/inventoryfields/Subunit.php]: oh thx :)
I'll change only author details: I subscribe YetiForce Public License 3.0

Comment (davide-alghi) [reply modules/Vtiger/inventoryfields/Subunit.php]: done

**Comment by davide-alghi** [comment]
If my Notes in PR will be taken into account, it's to highlight that changes will affect Textparser functions for lineitems tables. Unit and Subunit types are not enumered. Now unit and subunit are displayed into PDF because Value type is into foreach(<types>).

Anyway, this is currently a problem for Unit type field when user creates a custom inventory module or, for an existing inventory module, remove and then add again Unit type field: this re-added will not be rendered into PDF.

Affected textparser functions

- ProductsTableRelatedModule
- ProductsTableNew
- ProductsTableLongVTwoLang
- ProductsTableLongVersion
- ProductsTableCorrectingBefore
- ProductsTable
- DynamicInventoryColumnsTable (maybe, I didn't investigate)

**Pull request merged**